docs/sources/reference/components/prometheus/prometheus.exporter.snmp.md

```diff
@@ -127,102 +137,9 @@ debug information.
 `prometheus.exporter.snmp` does not expose any component-specific
 debug metrics.
 
-## Example
+## Examples using the targets argument (recommended)
```

Comment:
```suggestion
## Examples using the targets argument

The following examples show you how to use `prometheus.exporter.snmp`.
```

I'd suggest we remove `recommended` here. There's no alternative deprecated example provided... 

We should try to add at least a sentence of text between headings.

Reply:
There is indeed a deprecated example:

> `## Example using the target block (deprecated)`

Reply:
Oh.. ooops. missed that. The comment about collapsed section similar to the linked example still stands though.  I'd like to see the examples only show the current config (vs saying "recommended") and we keep the deprecated stuff documented during the period it's still in Alloy... but tuck it away under a collapsed section.